### DISCUSSION TITLE
vitepress的样式主题支持 and 更自由的样式覆盖支持

### DISCUSSION THREAD

**Discussion by qweasdzxcpkh** [imzbf/md-editor-v3#601] [Ideas]
# 需求描述

1. 现在vue团队已全力维护vitepress而不是vuepress了，在previewTheme中只有vuepress，但是它跟vitepress渲染的markdown样式其实是不一样的。所以希望可以在previewTheme中支持vitepress类型的theme
2. 看了下文档，貌似只支持样式变量覆盖去自定义样式，但是有些地方不能依靠样式变量实现，比如vitepress的blockquote缩进是用margin-left实现的，但是md-editor-preview blockquote则是padding-left。但是我没办法在引用editor样式的同时去除preview的样式，因为他们打包在同一个文件了（md-editor-v3/lib/style.css）
![image](https://github.com/user-attachments/assets/b9a5160e-9746-4996-bfbe-42e52c825741)
![image](https://github.com/user-attachments/assets/b4b7f016-3c10-4746-a611-30bbe0f67763)

**Comment by imzbf** [comment]
在5.0的主要功能开发完成后，我会考虑去搜集一份vitepress的样式，目前没有其他的维护者，所以可能需要一段事件
